app/src/main/java/ayds/spotisong/aurora/otherdetails/model/repository/OtherInfoRepositoryImp.kt

```diff
@@ -0,0 +1,32 @@
+package ayds.spotisong.aurora.otherdetails.model.repository
+
+import ayds.spotisong.aurora.otherdetails.model.repository.external.OtherInfoExternalService
+import ayds.spotisong.aurora.otherdetails.model.repository.local.DataBase
+import ayds.spotisong.aurora.song.model.Song
+
+private const val EXISTE_EN_DB_TAG = "[*]"
+
+internal class OtherInfoRepositoryImp(val service : OtherInfoExternalService, val dataBase: DataBase) : OtherInfoRepository{
+
+    override fun getArtistData(song: Song): ArtistWrapper {
+        val artistWrapper: ArtistWrapper
+        val artistName = song.artistName
+        var artistBio = dataBase.getBio(artistName)
+        var artistImagePath = dataBase.getImageUrl(artistName)
```

Comment:
No hace falta q sea variable 
```suggestion
        val artistImagePath = dataBase.getImageUrl(artistName)
```